docs/API.md

```diff
@@ -21,11 +21,13 @@ There are a number of methods that a DB storage backend should implement:
     * .sessionTokenWithVerificationStatus(tokenId)
     * .sessionWithDevice(tokenId)
     * .deleteSessionToken(tokenId)
-    * .verifyToken(tokenVerificationId, accountData)
 * Key Fetch Tokens
     * .createKeyFetchToken(tokenId, keyFetchToken)
     * .keyFetchToken(id)
+    * .keyFetchTokenWithVerificationStatus(tokenId)
```

Comment:
Will we ever want to look up a keyFetchToken without also knowing its verification status?

I'm happy to keep separate `keyFetchToken` and `keyFetchTokenWithVerificationStatus` methods for symmetry with the sessiontoken ones, mostly just curious...